mkcloudruns/README.md

```diff
@@ -0,0 +1,156 @@
+MkCloud Runs
+------------
+
+# Description
+
+Run multiple copies of various scenarios of SUSE OpenStack Cloud product
+using mkcloud ([automation repository](https://github.com/SUSE-Cloud/automation)).
+
+This repository is basically a wrapper which automates the boilerplate required
+to start mkcloud with the right configuration settings. Even for non-beginners
+this boilerplate wrapper script should be really handy due to the overwhelming
+amount of configuration options and environment variables used by mkcloud :|.
+
+- Inspiration:
+    > I'm just too lazy.
+
+<<<<<<< HEAD
+# Usage:
+
+Simply copy the template folder and create a new one with the given number.
+Check out the scripts, I will update the readme once I get more time for the same!
+Please contact me on IRC freenode on #openstack/#openstack-doc/#opensuse channels if in doubt
+or write me an email ...
+
+``$ ./deployClouds``
+
+=======
+- The perfect world:
+    > We do not need these scripts!
+>>>>>>> 6acf6db... Stub temp.
+
+# Deploying SUSE CLoud
+
+Follow these steps to deploy the required SUSE Cloud setup.
+
+## Initial Setup
+
+* Clone the repository
+  * These wrappers would eventually be part of automation repository, so I am trying to serve both worlds here ... use your common sense ;).
+* Setup up libvirt, KVM,LVM as per the automation repo, follow (this link)[https://github.com/dguitarbite/automation/blob/master/docs/mkcloud.md]
```

Comment:
Wrong link syntax here. I suggest viewing the rendered HTML version at https://github.com/dguitarbite/automation/tree/master/mkcloudruns to check for other errors.